src/main/java/core/basesyntax/Factorial.java

```diff
@@ -1,10 +1,13 @@
 package core.basesyntax;
 
 /**
- * <p>Реализуйте метод getFactorial(int number) который вовзращает факториал числа.</p>
+ * Реализуйте метод getFactorial(int number) который вовзращает факториал числа.
  */
 public class Factorial {
     public int getFactorial(int number) {
-        return 0;
+        if (number == 0) {
```

Comment:
move the number == 1 check to this statement

Reply:
The number == 1 check is moved to the statement. Thank you.